docs/modules/airflow/examples/example-airflow-kubernetes-executor-s3-xcom.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: airflow.stackable.tech/v1alpha1
+kind: AirflowCluster
+metadata:
+  name: airflow
+spec:
+  image:
+    productVersion: 2.9.3
+  clusterConfig: {}
+  webservers:
+    envOverrides: &envOverrides
+      AIRFLOW__CORE__XCOM_BACKEND: airflow.providers.common.io.xcom.backend.XComObjectStorageBackend
+      # The connection id is obtained from the user part of the url that you will provide
+      AIRFLOW__COMMON_IO__XCOM_OBJECTSTORAGE_PATH: s3://minio@como-data-platform/airflow/xcom
```

Comment:
Should we replace the specific data-platform with something more "generic"?

Reply:
Good catch! bc52906

Reply:
Ah, I did see it but thought it was just random :D